jobs/admin.py

```diff
@@ -1,3 +1,19 @@
 from django.contrib import admin
 
-# Register your models here.
+from .models import Company, Job
+
+
+def make_published(modeladmin, request, queryset):
+    queryset.update(published=True)
+    for job in queryset:
+        job.publish()
+    queryset.update()
+make_published.short_description = "Mark selected as published"
+
+class JobAdmin(admin.ModelAdmin):
+    list_display = ['title', 'company', 'reviewer', 'published']
+    ordering = ['title']
+    actions = [make_published]
+
```

Comment:
Make post_date readonly?  Since the post date is set when we publish a job, we don't necessarily want to allow admin users to set it manually
```readonly_fields=('post_date',)```